src/variables/body/search.tsx

```diff
@@ -16,14 +16,11 @@ export class Search extends Widget {
 
     const layout = new PanelLayout();
     this.layout = layout;
-    this.scope = new ScopeSearch(model);
     this.search = new SearchInput(model);
 
-    layout.addWidget(this.scope);
-    layout.addWidget(this.search);
+    // layout.addWidget(this.search);
```

Comment:
This comment can be removed

Reply:
I actually should remove that file cause with react-inspector we don't need search. (for now)

Reply:
Yes good point.